charts/script-exporter/templates/deployment.yaml

```diff
@@ -72,4 +74,4 @@ spec:
             defaultMode: 0777
         {{- if .Values.volumes }}
         {{- toYaml .Values.volumes | nindent 8 }}
-        {{- end }}
+        {{- end }}
```

Comment:
Can you re-add the deleted newline please.

Reply:
It's not deleted line you had empty whitespace after it was fixed automatically by the `vscode` editor